Resources/Prototypes/Maps/weedchurch.yml

```diff
@@ -0,0 +1,20 @@
+- type: gameMap
+  id: WeedChurch
+  mapName: 'Weed Church'
+  mapPath: /Maps/weedchurch.yml
+  minPlayers: 0
+  maxPlayers: 100
+  stations:
+    WeedChurch:
+      stationProto: StandardFrontierStation
+      components:
+        - type: StationNameSetup
+          mapNameTemplate: '{0} Frontier Outpost {1}'
+          nameGenerator:
+            !type:NanotrasenNameGenerator
+            prefixCreator: '14'
+        - type: StationJobs
+          overflowJobs:
+            - Passenger
```

Comment:
this makes the station have infinite open job slots, and round start even. I'm not necessarily opposed to having new round start locations, but we should hold off for now until we can chat on discord about what a station might need to be a round start spawn point

Reply:
Was for testing purposes, im fine with no roundstart spawns. but yeah HMU and we can talk